src/features/device-state/routes/state-get-v3.ts

```diff
@@ -334,19 +342,25 @@ const getDevice = async (req: Request, uuid: string) => {
 	return device;
 };
 
-const getConfig = (device: AnyObject) => {
+export const getConfig = (
+	device: AnyObject | undefined,
+	application: AnyObject = device?.belongs_to__application[0],
+) => {
 	const config: Dictionary<string> = {};
 
 	// add any app-specific config values...
-	const userAppFromApi: AnyObject = device.belongs_to__application[0];
+
 	varListInsert(
```

Comment:
I think it'd make sense to use the same method for both missing app and missing device, eg
```ts
	if (application) {
		varListInsert(application.application_config_variable, config, rejectUiConfig);
	}